config/templates/cloud-config-controller

```diff
@@ -367,6 +313,10 @@ write_files:
     owner: root:root
     content: |
       #!/bin/bash -e
+      {{ if .UseCalico }}
+      /bin/bash /opt/bin/populate-tls-calico-etcd
+      /usr/bin/docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests {{.HyperkubeImageRepo}}:{{.K8sVer}} /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
why not `curl -XPOST` like the rest of the script? or maybe it's better change curl to kubectl call?